2021q2/freebsd-foundation.adoc

```diff
@@ -0,0 +1,94 @@
+=== FreeBSD Foundation
+
+Contact: Deb Goodkin <deb@FreeBSDFoundation.org>
+
+The FreeBSD Foundation is a 501(c)(3) non-profit organization dedicated to supporting and promoting the FreeBSD Project and community worldwide.
+Funding comes from individual and corporate donations and is used to fund and manage software development projects, conferences and developer summits, and provide travel grants to FreeBSD contributors.
+The Foundation purchases and supports hardware to improve and maintain FreeBSD infrastructure and provides resources to improve security, quality assurance, and software engineering efforts; publishes marketing material to promote, educate, and advocate for the FreeBSD Project; facilitates collaboration between commercial vendors and FreeBSD developers; and finally, represents the FreeBSD Project in executing contracts, license agreements, and other legal arrangements that require a recognized legal entity.
+
+Here are some highlights of what we did to help FreeBSD last quarter:
+
+==== COVID-19 Impact to the Foundation
+
+Like most organizations, our team continued to work from home.
+Our temporary ban on travel for staff members remains in effect, but continues to not affect our output too much, since most conferences are still virtual.
+As the world continues to open up we will re-evaluate the travel ban.
+We continued supporting the community and Project through our regular channels.
+
+==== Partnerships and Commercial User Support
+
+We help facilitate collaboration between commercial users and FreeBSD developers.
+We also meet with companies to discuss their needs and bring that information back to the Project.
+Our temporary travel ban stayed in effect during Q2, so we continued meeting with corporate users virtually.
+If things look good for in-person meetings in the fall, then we'll start incorporating those into an in-person and virtual meeting mix.
+
+==== Fundraising Efforts
+
+First, we’d like to say thank you to everyone who has given us a financial contribution this year!
+Last quarter we raised $70,410, which includes donations from organizations like Verisign, VMware, and Stormshield, as well as many individuals.
+
+We depend on these donations to fund our work supporting FreeBSD.
+Late last year we decided to put more funding into helping to improve FreeBSD.
+We hired a Sr. Software Developer to work on arm64 and a Project Coordinator to manage our projects and interface with the community.
+We also hired two of our part-time software developers full-time.
+The purpose of this was to provide more resources to step in to implement and improve major features in FreeBSD, review patches and bug reports, implement bug fixes, and support the security efforts.
+This keeps FreeBSD the innovative, secure, and reliable operating system you rely on.
```

Comment:
"reliable operating system you rely on" reads a bit awkward to me. Perhaps "... you use"?

Reply:
Will take a stab at an edit